projects/packages/backup/changelog/add-backup-ui-to-jetpack

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
Since the change in this PR doing this seems to have been reverted, do we still need a change entry for it?